pkgs/by-name/mo/monaspace/package.nix

```
@@ -3,34 +3,9 @@
   stdenvNoCC,
   fetchzip,
 }:
```

This function lets us effectively combine multiple derivations into one.

```suggestion
  symlinkJoin
}:
```